osu.Desktop/DiscordRichPresence.cs

```diff
@@ -114,6 +133,34 @@ private void updateStatus()
                 {
                     presence.Buttons = null;
                 }
+
+                if (!hideIdentifiableInformation && multiplayerClient.Room != null)
+                {
+                    MultiplayerRoom room = multiplayerClient.Room;
+                    presence.Party = new Party
```

Comment:
This presence must be updated on changes to multiplayer room, it cannot be left as follow-up effort, since it makes the feature completely broken. Subscribing to `MultiplayerClient.RoomChanged` is probably simplest.

Reply:
Oops, that's a little obvious. Should've caught that. :sweat_smile:

Thank you for the fix.